optimum/commands/export/neuron.py

```diff
@@ -60,12 +60,16 @@ def parse_args_neuron(parser: "ArgumentParser"):
     )
     optional_group.add_argument(
         "--auto_cast",
+        type=str,
         default=None,
+        choices=["None", "matmul", "all"],
```

Comment:
```suggestion
        choices=["none", "matmul", "all"],
```

Reply:
Corrected with changes in __main__ [here](https://github.com/huggingface/optimum-neuron/pull/55/commits/ff62fdef264728d1cddc339077e53863a0689035)